frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.selectors.ts

```diff
@@ -136,19 +152,23 @@ export const selectFeatureFlagInclusions = createSelector(
     if (!featureFlag || !featureFlag.featureFlagSegmentInclusion) {
       return [];
     }
-    return [...featureFlag.featureFlagSegmentInclusion]
-      .sort((a, b) => new Date(a.segment.createdAt).getTime() - new Date(b.segment.createdAt).getTime())
-      .map((inclusion) => {
-        return {
-          segment: inclusion.segment,
-          listType: inclusion.listType,
-          enabled: inclusion.enabled,
-        };
-      });
+    return featureFlag.featureFlagSegmentInclusion.map((inclusion) => ({
+      segment: inclusion.segment,
```

Comment:
why was the sorting stuff moved? It makes more sense here than in the selectSelectedFeatureFlag right?